### PR TITLE
bruce-binary/gamstop_api_v2

### PR DESCRIPTION
**Pull request by bruce-binary**

### CODE REVIEW

**Comment by raunakkathuria** [comment]
Please check failing tests

**Comment by bruce-binary** [comment]
@raunakkathuria hm. It's more of the CI issue. I'll take a look.

File: .travis.yml [hunk 1]
```diff
@@ -4,14 +4,13 @@ matrix:
...
-   - '5.14'
 before_install:
    - git config --global user.name "TravisCI"
    - git config --global user.email $HOSTNAME":not-for-mail@travis-ci.org"
 install:
    - cpanm --quiet --notest Dist::Zilla Dist::Zilla::App::Command::cover App::cpm Devel::Cover::Report::Codecov
-   - cpm install -w 2 --mirror=http://cpan.cpantesters.org/ -g $(dzil authordeps --missing)
-   - cpm install -w 2 --mirror=http://cpan.cpantesters.org/ -g $(dzil listdeps --author --missing)
+   - cpanm --quiet --notest --mirror=http://cpan.cpantesters.org/ $(dzil authordeps --missing)
+   - cpanm --quiet --notest --mirror=http://cpan.cpantesters.org/ $(dzil listdeps --author --missing)
```
Comment (bruce-binary) [inline .travis.yml]: whelp @raunakkathuria not sure why we use both `cpanm` and `cpm` but now the virtual machine doesn't have `cpm`, not could it find `5.14` perl package

Comment (raunakkathuria) [reply .travis.yml]: We should move away to CircleCi for this repo. This will do for now.

**Pull request merged**